server/server.js

```diff
@@ -2,56 +2,50 @@ const express = require('express');
 const bodyParser = require('body-parser');
 const cors = require('cors');
 
-// My cats resource
-const cats = [
-    {id: 1, name: "Zelda", age: 3},
-    {id: 2, name: "Tigerlily", age: 10},
-    {id: 3, name: "Rumble", age: 12}
+// Searches Resource
+const searchResults = [
+    {search: "cats", url:"https://bit.ly/3eOEoCW"},
+    {search: "dogs", url:"https://bit.ly/3n9mMo7"}, 
+    {search: "cars", url:"https://bit.ly/3piKCzx"},
+    {search: "planes", url: "https://bit.ly/3pgubnC"},
+    {search: "oranges", url: "https://bit.ly/3eJnDc0"},
+    {search: "apples", url: "https://bit.ly/32vOiUW"},
+    {search: "movies", url: "https://bit.ly/2Ir3H1Q"},
+    {search: "radios", url: "https://bit.ly/3kqx5Tb"},
+    {search: "shoes", url: "https://bit.ly/36trYwy"},
+    {search: "hats", url: "https://bit.ly/2UcrWn1"}
 ]
 
 const app = express();
 app.use(bodyParser.json())
 app.use(cors())
 
 app.get('/', (req, res) => {
-    res.send('Hello there!')
+    res.send('Hello user, welcome to Google!')
 })
 
-app.post('/', (req, res) => {
-    res.status(405).send('Not allowd!')
+app.get('/searches', (req, res) => {
+    res.send(searchResults)
```

Comment:
Again nice routing - future feature could be linking this with your front end fully